app/build.gradle

```diff
@@ -7,6 +7,14 @@ plugins {
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
+    viewBinding{
```

Comment:
Could you please add a space character before opening the braces?

Reply:
Added space.